pkg/machine/env/dir.go

```diff
@@ -56,7 +56,11 @@ func GetMachineDirs(vmType define.VMType) (*define.MachineDirs, error) {
 		return nil, err
 	}
 
-	rtDir = filepath.Join(rtDir, "podman")
+	// The runtime directory can be customized using the PODMAN_RUNTIME_DIR env variable
+	// Its default value will be podman
+	// When used by macadam it can be customized to use a different path like macadam
+	runtimeDir := GetRuntimeDir()
```

Comment:
This could/should also be named `GetRuntimeDirSuffix()`, I forgot to mention it in the previous review :-/

Apart from this, this looks good to me.

Reply:
Updated 👍